test/Manifest.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.2"
+julia_version = "1.7.1"
```

Comment:
@NLaws  I think I accidentally changed this to 1.7.1 -- is this an issue and if so can I manually change it back?

Reply:
the Manifest.toml was changed to your julia_version when you `Pkg.add`ed DataFrames and CSV to the test environment. I left a review comment about removing those dependencies so you should be able to revert the test/Project.toml and test/Manifest.toml back to their original states. But in general I don't think that it will be an issue but we should try to all use the latest stable version of Julia.